web/features.json

```diff
@@ -479,6 +479,12 @@
 				"introduced":9
 			},
 			{
+				"name":"Quick Armor Swapping",
+				"description":"Right clicking an armor stand while sneaking will swap all armor and your off hand item quickly.",
+				"image":"quick-armor-swapping.jpg",
+				"introduced":11
```

Comment:
This should include the new "contributed" entry, added in https://github.com/Vazkii/Quark/commit/723e261a2baeb4c860296b842ea9343e5fd9603f